fastlane/lib/fastlane/actions/notarize.rb

```diff
@@ -74,7 +74,7 @@ def self.notarytool(params, package_path, bundle_id, skip_stapling, print_log, v
         ] + auth_parts
 
         if verbose
-          submit_command << "--verbose"
+          submit_parts << "--verbose"
```

Comment:
this will add lots of debug logs to the response of the sh command, and the line 89:
```ruby
 notarization_info = JSON.parse(submit_response)
 ```
 will fail to parse a json due to the debug logs being part of the response

Reply:
The output of `man notarytool` shows
> --verbose   Enable streaming of DEBUG level logs to stderr.

I don't see any code in `sh_helper.rb` that includes `stderr` in the result of the `Actions.sh()` command run here on line 81, so adding `--verbose` should not affect the JSON parsing.